tests/kuttl/supplychain/templates-ytt/01-supply-chain.yaml

```diff
@@ -16,41 +16,41 @@
 apiVersion: carto.run/v1alpha1
 kind: ClusterSupplyChain
 metadata:
-  name: responsible-ops---consume-output-of-resources
+  name: responsible-ops---templates-ytt
 spec:
   selector:
-    integration-test: "templates-consume-output-of-resources"
+    integration-test: "templates-templates-ytt"
```

Comment:
nitpick- we can probably drop the first "templates" to have it match the name of the test, like the `templates-consume-output-of-components` test does. not a big deal though!